grpc/pbdirect/src/main/scala/sttp/tapir/grpc/protobuf/pbdirect/TapirProtobufPbDirect.scala

```diff
@@ -0,0 +1,19 @@
+package sttp.tapir.grpc.protobuf.pbdirect
+
+import _root_.pbdirect._
+import sttp.tapir.{Codec, CodecFormat, DecodeResult, EndpointIO, RawBodyType, Schema}
+
+trait TapirProtobufPbDirect {
+  def grpcBody[T](implicit codec: Codec[Array[Byte], T, CodecFormat.OctetStream]): EndpointIO.Body[Array[Byte], T] =
```

Comment:
use implicit `PBMessageWriter ` and `PBMessageReader ` here directly for better error messages